go.mod

```diff
@@ -37,36 +37,36 @@ require (
 	github.com/alecthomas/kingpin/v2 v2.4.0 // replaced
 	github.com/alicebob/miniredis/v2 v2.32.1
 	github.com/andybalholm/brotli v1.1.0
-	github.com/aquasecurity/libbpfgo v0.5.1-libbpf-1.2
+	github.com/aquasecurity/libbpfgo v0.6.0-libbpf-1.3
```

Comment:
Careful here. Better move this one to its own PR, it usually wants updates to libbpf itself.